app/views/api/integrations/apicast/configuration_driven/_apicast.html.slim

```diff
@@ -34,6 +34,13 @@ section.Configuration class=('Configuration--is-promoted' unless deployment_opti
         dt.u-dl-term Secret Token
         dd.u-dl-definition = @proxy.secret_token
 
+    div
+      = semantic_form_for @proxy, url: admin_service_integration_path(@service) do |f|
+        = f.buttons class: "buttons buttons-inline" do
+          = f.hidden_field :lock_version
+          = f.button "Promote v. #{@show_presenter.last_sandbox_config.version+1} to Staging",
```

Comment:
How does this work with Service Mesh?

Normally we show only one button for it, there is no deploy to Staging